Tests/Data

```diff
@@ -1 +1 @@
-Subproject commit 3087421f7ad138c72be7ee85f6872925da48ba7b
+Subproject commit 28d74b31d74aa5f03a418cc5a11f33cd1e2d7662
```

Comment:
In the Tests/Data could the Elliptic/cube_1x1x1_NeumannBC/ could you please cleanup the vtu files? There are unnecessary data arrays in the input vtu and output vtus. (Alternatively one could refer to the original input vtu (without copy) which is one level up).